### ISSUE TITLE
Native indexing is including all files under .git/ on Windows

### ISSUE THREAD

**Issue by mkleehammer** [bbatsov/projectile#663]
I'm using emacs 24.4.1 on OS X and Windows.  On Windows I've recently noticed that all .git/ files (including the objects directory!) are being indexed on Windows.  I don't see this on OS X which is also using native indexing and shares the same emacs init files.  

I didn't make any recent changes and both are running version 20150223.8 from MELPA.

Suggestions on troubleshooting this?

The only customizations I have are:

```elisp
(use-package projectile
  :ensure t
  :diminish projectile-mode
  :config
  (progn
    (setq projectile-enable-caching t
          projectile-switch-project-action 'projectile-dired
          projectile-globally-ignored-files
          (append projectile-globally-ignored-files '(".DS_Store" ".gitignore" "*.pyo" "*.dll" "*.pdf" "*.exe" "*.pyc" "*.elc")))
    (add-to-list 'projectile-other-file-alist '("html" "js" "py"))
    (add-to-list 'projectile-other-file-alist '("js" "html" "py"))
    (add-to-list 'projectile-other-file-alist '("py" "html" "js"))))
```

In one of my projects I have a .projectile file and in another I do not.  Both are indexing all files under .git.

**Comment by mkleehammer** [comment]
Actually it looks like no filtering is occurring.  The file list (C-c p f) includes *.exe files and others from the globally ignored files list above.  I know this was working in the past.

**Comment by mkleehammer** [comment]
I'm going to close this because I've moved to "alien" indexing.  (BTW, I would have assumed "native" meant native to the OS and I've seen others.  What about "internal vs external" or "emacs vs external"?)

I didn't try it since the documentation indicate it doesn't work well on Windows.  When I noticed the setting for git ls-files I tried it and it is great.  I assume it is non-git projects that don't work well?
